pkg/utils/utils.go

```diff
@@ -36,3 +38,22 @@ func IsNonEmptyFile(dir, file string) bool {
 	}
 	return !info.IsDir() && info.Size() > 0
 }
+
```

Comment:
thanks for the PR! Can we add unit tests for these please?

Reply:
Done!